### DISCUSSION TITLE
Migrate to http from Dio

### DISCUSSION THREAD

**Discussion by CHRISTOPANANJICKAL** [CHRISTOPANANJICKAL/fast_cached_network_image#29] [Announcements]
## 👋 Hii folks!
   I would like to know your opinion on migrating this package to [http](https://pub.dev/packages/http) from [Dio](https://pub.dev/packages/dio)

**Comment by jnelle** [comment]
Stay with http, you don't have add an other dependency, just keep this package as small and simple as possible
